src/sources/soundsourcemodplug.cpp

```diff
@@ -133,8 +133,6 @@ SoundSource::OpenResult SoundSourceModPlug::tryOpen(
     m_pModFile = ModPlug::ModPlug_Load(m_fileBuf.constData(),
             m_fileBuf.length());
     if (m_pModFile == nullptr) {
-        // an error occurred
-        t.cancel();
```

Comment:
This was used to not have a log entry of the "SoundSourceModPlug::open()" timer. 
Was it intended to have now always a timer entry? 

My original suggestion was to keep cancel() but reset the optional during cancel. This has the same effect.

Reply:
Ah yeah, I misunderstood the code. I'll revert.